### DISCUSSION TITLE
PydanticUserError: in from bertopic import BERTopic

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#1805] [Q&A]
Hi, There is some overlap in the model installation. I am getting this error when I am doing "from bertopic import BERTopic" in my jupyter notebook as "PydanticUserError: Field 'name' defined on a base class was overridden by a non-annotated attribute. All field definitions, including overrides, require a type annotation."  Please help!

**Comment by MaartenGr** [comment]
Thanks for sharing the issue. Could you provide the **full** code and error message? Unfortunately, it is not entirely clear to me what you mean by _some overlap in the model installation.

Also, did you try a clean install and environment?

**Comment by Nee-DS** [comment]
Thanks, Maarten, I was able to solve this issue with a clean install and also this was related to the Pyinstaller version, which got updated recently.
